evap/rewards/models.py

```diff
@@ -51,7 +51,7 @@ class RewardPointGranting(models.Model):
     user_profile = models.ForeignKey(UserProfile, models.CASCADE, related_name="reward_point_grantings")
     semester = models.ForeignKey(Semester, models.PROTECT, related_name="reward_point_grantings")
     granting_time = models.DateTimeField(verbose_name=_("granting time"), auto_now_add=True)
-    value = models.IntegerField(verbose_name=_("value"), default=0)
+    value = models.IntegerField(verbose_name=_("value"), default=0) # Might consider enforcing MinValue in the future
```

Comment:
What is the status on this? Before merging, the comment should probably be more concrete as to why we are not enforcing it, or implement the constraint :)

Reply:
It used to be the default value, but is apparently not wanted. Therefore, adding this constraint could break the database.

Reply:
@janno42 Do we have `RewardPointGrantings` with `value==0` on production? I'd expect no, but just to be sure. If we don't have any, we can add the constraint here.

Reply:
We don't have any instances with `value = 0` in the production database, so we can add this constraint here. I'd vote in favor of adding it :)